components/accelerator/nvidia/fabric-manager/component.go

```diff
@@ -76,7 +75,12 @@ func (c *component) States(ctx context.Context) ([]components.State, error) {
 		}, nil
 	}
 
-	defaultConn := systemd.GetDefaultDbusConn()
+	cctx, cancel := context.WithCancel(c.rootCtx)
+	defer cancel()
+	defaultConn, err := pkg_systemd.NewDbusConn(cctx)
```

Comment:
Eventually we should remove the dependency on dbus..

Reply:
yes, https://github.com/leptonai/gpud/issues/594